ui/src/views/WorkerManager/WMViewWorkerPools/WMWorkerPools.graphql

```diff
@@ -1,14 +1,9 @@
 query workerPools {
   WorkerManagerWorkerPoolSummaries {
-    workerPool
-    pendingCapacity
-    runningCapacity
+    workerPoolId
+    providerId
+    description
+    owner
     pendingTasks
```

Comment:
I think it would be beneficial to add a newline before `pendingTasks` to warn developers that this will trigger an extra request.

Reply:
Newline itself doesn't explain anything. I can add a comment to that effect.

If there is no way to automate such newlines, it means it's a matter of taste. Maybe we could omit preferential suggestions in code reviews?

Reply:
The reason why we are adding a newline instead of a comment is because GraphQL self documents itself based on the comments defined above the properties in the graphql definition file. For example:

![Screen Shot 2019-06-06 at 1 10 29 PM](https://user-images.githubusercontent.com/3766511/59052187-7f8fa580-885c-11e9-9a5a-a0eb24a03313.png)





Adding information about an extra request didn't seem like a good place to put it as a comment at the time since that didn't really define the purpose of that property. That being said, removing all of the newlines and replacing them with a comment that would help make it clear to developers which properties would trigger an extra request is also another option. I don't mind which method we take but I think we should  aim for consistency. Thoughts?

Reply:
My thoughts are:

* Personally, I don't understand newlines. The reason for the newline never occurred to me until you told me why it's there :) I suspect I'm not alone. I view comments as easier to understand than formatting

* Why don't we automate this? I would like our reviews to be more about finding actual bugs, errors in the logic, security problems etc. Why don't we use a style guide and a linter for the rest?

Reply:
> Personally, I don't understand newlines.

I don't as well. Let's change things to use comments. But let's change it everywhere. Maybe we can create an issue to track this?

> Why don't we automate this?

I wish everything was automated :) I don't have the bandwidth to work on this right now but feel free to create an issue or something for this.

Reply:
>  Why don't we use a style guide and a linter for the rest?

We should. I will update the README to have a styleguide section. We could have the newline thing there as a start. Hope that helps.

Reply:
> Maybe we can create an issue to track this?

Sure, I can create an issue

> I don't have the bandwidth to work on this right now but feel free to create an issue or something for this.

Sure. Although sometimes it seems to me you could actually save yourself a lot of time by automating all these things